### ISSUE TITLE
License rethink

### ISSUE THREAD

**Issue by Tea23** [haleyjd/sin-ex-game#1]
The current license does not permit much in the way of community development. Re-licensing under the GPL or MIT should be considered to allow a wider community development scope. It would, with the current license, be quite difficult for a community developed fork project to get off the ground since doing something like that would require the express permission of Night Dive. Any forks would also need the same license, and that leaves us in a tricky position should Night Dive cease to exist.

What do you think, sirs?

**Comment by jasonmerc** [comment]
I agree.  What is and is not allowed should be more clearly spelled out.  It's not "open source" as it is right now, which begs the question... why release it in the first place?

**Comment by hogsy** [comment]
I'm not really providing anything useful here except to +1 the suggestion of reviewing the licensing. It was dissapointing that this isn't under a very liberal license - particularly given all this is right now is just the game code (though it would be great to see you guys open-source the engine too, *wink wink*).

I totally get that there is an official SiN remaster in the works, which I suppose might be why this currently isn't the case, but at the same time, if Nightdive really want to contribute towards preserving games such as SiN I think it's worth seriously considering making the code available under an open-source license such as GPL or MIT.

If it's a matter of manpower, I'm very much happy to prepare the code for such a release for absolutely nothing and you're more than welcome to contact me.
